src/shapes/IText/IText.ts

```diff
@@ -482,9 +482,9 @@ export class IText<
    * Used for the drag/drop effect.
    * If contextTop is not available, do nothing.
    */
-  renderCursorAt(selectionStart: number) {
+  renderCursorAt(selectionStart: number, ctx = this.canvas!.contextTop) {
```

Comment:
I don't like this signature and prefer ctx to be the first param and mandatory to align with the rest of the methods.
This is a minor breaking change because it is used in the codebase only once